sklearn/metrics/cluster/tests/test_common.py

```diff
@@ -123,11 +123,14 @@ def test_normalized_output(metric_name):
     assert not (score < 0).any()
 
 
+SUPERVISED_UNSUPERVISED_METRICS = {**SUPERVISED_METRICS,
```

Comment:
Do sets not get ordered by insertion like dicts do in py 3.6?

Reply:
Sets are still unordered. In this case, `SUPERVISED_UNSUPERVISED_METRICS` is a dictionary. 

Looking over this, `SUPERVISED_UNSUPERVISED_METRICS` is not needed, since the dictionary is ordered. This change has been reverted.

Reply:
has it been reverted @thomasjpfan ?